README.md

```diff
@@ -16,43 +16,32 @@ You must set the following environment variables before running the program:
 * `AWS_SECRET_ACCESS_KEY` The secret key for your AWS account.
 * `AWS_DEFAULT_REGION` The default region to use, e.g. `us-east-1`.
 
-A python development environment is required. It is recommended to create a python virtual environment and 
-install dependencies from requirements.txt .e.g.
- 
-    python3 -m venv venv
-    source venv/bin/activate
-    pip install -r requirements.txt
-    
-This environment should be activated when running any of the tools, e.g.
-
-    source venv/bin/activate
-
 ## econ_dashprop
 
-econ_dashboard string properties against versions of articles within the eLife Continuum Dashboard.
-An example use for this is updating the publication-status property of an article to indicate it should be temporarily hidden.
+`econ_dashboard` string properties against versions of articles within the eLife Continuum Dashboard.
```

Comment:
this doesn't make sense to me

Reply:
I think the second line refers to the example below it, the `publication-status hidden` results in sending the `publication-status` value of `hidden` onto the `event-property-incoming-queue` for article `00288`, version `1`. That message is then picked up by the dashboard, which results in hiding version 1 of `00288` from the current list of articles that are errored or in progress. I do this occassionally for Production, and I use the example from the readme to remind me how to do it. The wording probably doesn't make sense unless you already know what it's describing. Are you able to improve it for better understanding, if I explained it better in the above?

Reply:
just took a look at the code and something was lost in the copy+paste to the readme it seems. It makes more sense now.

Thanks for the explanation, @gnott , I've added it beneath the example